migrations/2_deploy_contracts.js

```diff
@@ -22,7 +22,12 @@ if (typeof process.env.SGX_MODE === 'undefined' || (process.env.SGX_MODE != 'SW'
     EnigmaContract = require('../build/enigma_contracts/Enigma.json');
 }
 const EnigmaTokenContract = require('../build/enigma_contracts/EnigmaToken.json');
-const provider = new Web3.providers.HttpProvider('http://localhost:9545');
+
+const ethHost = (typeof process.env.ETH_HOST === 'undefined') ? 'localhost' : process.env.ETH_HOST;
+const ethPort = (typeof process.env.ETH_PORT === 'undefined') ? '9545' : process.env.ETH_PORT;
+
```

Comment:
Note that you can express this much more elegantly with
```js
const ethHost = process.env.ETH_HOST || 'localhost'
const ethPort = process.env.ETH_PORT || '9545'
```
Same story with the diff in test/api.test.js
(I've used this pattern in #35 a lot)

Reply:
Great observation, @reuvenpo, thank you!
Improved in 14f12e4f0f0345077a4c46c2a11422f91620fb7a